src/app/search-bar/search-bar.component.ts

```diff
@@ -14,7 +14,10 @@ import * as speechactions from '../actions/speech';
 @Component({
   selector: 'app-search-bar',
   templateUrl: './search-bar.component.html',
-  styleUrls: ['./search-bar.component.css']
+  styleUrls: ['./search-bar.component.css'],
+  host: {
+          '(document:click)': 'sugClose($event)'
+        }
```

Comment:
Please follow the coding style properly.

Reply:
I have indented the code, what should I do now ?